sbag/generators/javascript/__init__.py

```diff
@@ -62,6 +77,24 @@ def get_path_parameters(endpoint):
     return ret
 
 
+def get_path_parameters2(endpoint):
+    ret = ''
+    if '{' in endpoint.path and '}' in endpoint.path:
+        for param in endpoint.parameters:
+            ret = ret + ' + ' + param + ' + ' + "'/'"
+        return ret
+    else:
+        return ret
+
+
+def get_custom_path_imports(path):
+    ret = set()
+    for endpoint in path:
+        if isinstance(endpoint.post_type, Entity):
+            ret.add(endpoint.post_type)
+    return ret
```

Comment:
You have this already in controller_imports variable
controller_imports[path_name] returns list of all entities you need to import for that controller

Reply:
Yes it does, but the way it is implemented, it includes more entities than I need

Reply:
Okay, it's fine then